.github/CHANGELOG.md

```diff
@@ -233,7 +233,12 @@
 1. [MISC] Added custom loading tips - @Armankir (Arman#5297)
 1. [MCDU] Revised Reserve Weight Calculation and Default weight rule-set added - @Lucky38i (Lucky38i)
 1. [RMP] Added integral illumination to RMP volume knobs - @ImenesFBW (Imenes)
+<<<<<<< HEAD
```

Comment:
You've accidentally comitted the merge markers here.

Reply:
solved :)